fili-core/src/main/java/com/yahoo/bard/webservice/table/resolver/DataSourceConstraint.java

```diff
@@ -160,4 +161,26 @@ public DataSourceConstraint withMetricIntersection(Set<String> metricNames) {
                 apiFilters
         );
     }
+
+    /**
+     * Build a constraint which should filter away no part of a given table.
+     *
+     * @param table  The table whose dimensions and metrics are to be queried
+     *
+     * @return a constraint which should provide no restrictions
+     */
+    public static DataSourceConstraint emptyConstraint(PhysicalTable table) {
```

Comment:
A little bit confusing, since it constraints to all table instead of no constraint at all.